chapters/en/chapter4.md

```diff
@@ -80,95 +84,92 @@ models, so we can create training data to teach a model to recognize them as
 
 <exercise id="4" title="Creating training data (2)">
 
-Let's use the match patterns we've created in the previous exercise to bootstrap
-a set of training examples. A list of sentences is available as the variable
-`TEXTS`.
+After creating the data for our corpus, we need to save it out to a `.spacy`
+file. The code from the previous example is already available.
 
-- Create a doc object for each text using `nlp.pipe`.
-- Match on the `doc` and create a list of matched spans.
-- Get `(start character, end character, label)` tuples of matched spans.
-- Format each example as a tuple of the text and a dict, mapping `"entities"` to
-  the entity tuples.
-- Append the example to `TRAINING_DATA` and inspect the printed data.
+- Instantiate the `DocBin` with the list of `docs`.
+- Save the `DocBin` to a file called `train.spacy`.
 
 <codeblock id="04_04">
 
-- To find matches, call the `matcher` on a `doc`.
-- The returned matches are `(match_id, start, end)` tuples.
-- To add an example to the list of training examples, you can use
-  `TRAINING_DATA.append()`.
+- You can initialize the `DocBin` with a list of docs by passing them in as the
+  keyword argument `docs`.
+- The `DocBin`'s `to_disk` method takes one argument: the path of the file to
+  save the binary data to. Make sure to use the file extension `.spacy`.
 
 </codeblock>
 
 </exercise>
 
-<exercise id="5" title="The training loop" type="slides,video">
+<exercise id="5" title="Configuring and running the training" type="slides">
 
-<slides source="chapter4_02_training-loop" start="39:00" end="42:25">
+<slides source="chapter4_02_running-training">
 </slides>
 
 </exercise>
 
-<exercise id="6" title="Setting up the pipeline">
+<exercise id="6" title="The training config">
 
-In this exercise, you'll prepare a spaCy pipeline to train the entity recognizer
-to recognize `"GADGET"` entities in a text – for example, "iPhone X".
+TODO (multiple choice)
```

Comment:
How about a "select the false option" kind of question here?

`Which of the following is not true about the training config files?`

Suggestions for true options:
- Allows you to configure the training process and hyperparameters
- Helps to make your training more reproducible
- Defines the pipeline's components and their settings
- Defines how to initialize the nlp object

Suggestions for false options:
- Helps you load your trained pipeline correctly into spaCy
- Includes the paths to your training and evaluation data 
- Defines the configuration to package your pipeline

Reply:
I like that style of question! 👍 The false options are tricky, though: _technically_ someone could argue that both are correct – the config does help you load the pipeline into spaCy (since it's used to set up the `nlp` object when you load a pipeline from disk). It definitely doesn't include packaging settings but it does influence the packaged pipeline so people might still be confused by that 🤔

Reply:
Oh, I can see that... How about "Creates an installable Python package with your pipeline" for a false alternative?